.github/tests/.env

```diff
@@ -28,10 +28,10 @@ DEFAULT_SMTP_PORT=
 DEFAULT_SMTP_TLS=
 DEFAULT_SMTP_NO_AUTH=
 
-USER_CONFIG_TYPE=DB
-USER_SETTINGS_DIR=/tmp
-ATTACHMENT_DIR=/tmp
-APP_DATA_DIR=/tmp
+USER_CONFIG_TYPE=file
```

Comment:
Are you sure the github tests are not working with db?

Reply:
Updated